template.json

```diff
@@ -0,0 +1,65 @@
+{
+  "package": {
+    "scripts": {
+      "start": "react-scripts start",
+      "build": "react-scripts build",
+      "build:staging": "env-cmd -f .env.staging npm run build",
+      "test": "react-scripts test",
+      "eject": "react-scripts eject",
+      "check-package-updates": "npx npm-check-updates",
+      "prepare": "husky install"
+    },
+    "dependencies": {
+      "@testing-library/jest-dom": "^5.14.1",
+      "@testing-library/react": "^12.0.0",
+      "@testing-library/user-event": "^13.2.1",
+      "@types/jest": "^27.0.1",
+      "@types/node": "^16.7.13",
+      "@types/react": "^17.0.20",
+      "@types/react-dom": "^17.0.9",
+      "typescript": "^4.4.2",
+      "web-vitals": "^2.1.0",
+      "classnames": "^2.3.1",
+      "env-cmd": "^10.1.0",
+      "react": "^17.0.2",
+      "react-dom": "^17.0.2",
+      "react-router-dom": "^5.3.0",
+      "react-scripts": "4.0.3"
```

Comment:
is there a reason we use version 4?
latest is the version 5 I guess.

Reply:
Copied this part from a project, so no reason. I checked the official template, looks like we don't even need to include these at all. I'll check again

Reply:
I'll remove these as we decided on the meeting